README.md

```diff
@@ -1,45 +1,71 @@
 # conjur-oss-suite-release
-Latest stable releases of the Conjur OSS suite
+
+Building the official releases of the [Conjur OSS suite](https://cyberark.github.io/conjur)
 
 **THIS REPO IS UNDER CONSTRUCTION, AND THERE ARE NO SUITE RELEASES AVAILABLE YET**
 
 _For more info on the planned work, please see our [design doc](https://github.com/cyberark/conjur/blob/master/design/oss_suite_release.md)
 and monitor the [github issues](https://github.com/cyberark/conjur-oss-suite-release/issues)._
 
+Once we have a first release with release notes, we'll be adding a link to the
```

Comment:
I guess we have a chicken and the egg issue here, as we don't have a public URL yet?

Reply:
yup